build.sbt

```diff
@@ -55,9 +55,12 @@ val commonSettings = Seq(
       </developers>
 )
 
+lazy val root = project.in(file("."))
+  .aggregate(`gremlin-scala`, macros)
+
```

Comment:
Though the build works fine without the declared root project I do find utility in it for with it, when you are in an `sbt` session and type `projects` the following information is output:
```
> projects
[info] In file:/Projects/gremlin-scala/
[info]     gremlin-scala
[info]     macros
[info]   * root
> 
```
Without it the following is output:
```
> projects
[info] In file:/Projects/gremlin-scala/
[info]   * default-28afd7
[info]     gremlin-scala
[info]     macros
>
```